dnscrypt-proxy/odoh_test.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	hpke "github.com/cisco/go-hpke"
+	godoh "github.com/cloudflare/odoh-go"
+	"testing"
+)
+
+func TestInterop(t *testing.T) {
```

Comment:
@jedisct1: I kept this here to make sure the client is at least correct. Would you prefer we implement server support in odoh.go and keep this test? If not, how would you prefer we test odoh.go?

Reply:
Functional tests.

The `.ci` directory has some of these; a script dedicated to oDoH can be added. No need to reimplement the server part, it can just install an existing server implementation for testing.

Reply:
That works for me :-) I'll remove this unit test, the odoh-go dependency, and put together a functional test.